.github/workflows/build_python.yml

```diff
@@ -17,18 +17,26 @@
 # under the License.
 #
 
+# According to https://infra.apache.org/github-actions-policy.html,
+# all workflows SHOULD have a job concurrency level less than or equal to 15.
+# To do that, we run one python version per cron schedule
 name: "Build / Python-only (master, PyPy 3.9/Python 3.10/Python 3.12)"
 
 on:
   schedule:
     - cron: '0 15 * * *'
+    - cron: '0 17 * * *'
+    - cron: '0 19 * * *'
```

Comment:
Although we can have multiple files like `build_python3.10.yml`, `build_python3.12.yml`, .... , I guess you prefer to keep this in a single file in order to avoid `Duplication`, @HyukjinKwon . 

Please let me know if we can duplicate this file simply per Python version.

Reply:
I'm fine either way 👍